### ISSUE TITLE
Google Analytics

### ISSUE THREAD

**Issue by mrgoodboy** [YaleSTC/YalePublic-ios#9]
Once everything Google Analytics is merged into the code base, add the following lines to your VC. 

If the VC extends UIViewController, set it to extend GAITrackedViewController instead and add the following lines to .m file:

- (void)viewDidAppear:(BOOL)animated
{
    [super viewDidAppear:animated];
    id<GAITracker> tracker = [[GAI sharedInstance] defaultTracker];
    [tracker sendView:@"SCREENNAME"]; //this should be in the format News VC, Photo VC etc
}

If your VC extends anything other than UIViewController, add these lines to the .m file

#import "GAI.h"
#import "GAIFields.h"
#import "GAIDictionaryBuilder.h"
...

- (void)viewDidAppear:(BOOL)animated
{
    [super viewDidAppear:animated];
    id tracker = [[GAI sharedInstance] defaultTracker];
    [tracker set:kGAIScreenName
           value:@"SCREENNAME"]; //this should be in the format News VC, Photo VC etc
    [tracker send:[[GAIDictionaryBuilder createScreenView] build]];
}

Peace
